bedrock/settings/base.py

```diff
@@ -1,3 +1,6 @@
+#!/usr/bin/env python
```

Comment:
It is for the currency symbols. When running the tests it fails. Seeing that having the symbols here is not going to be of much use, I am going to remove them and then these lines as well.